buildcontext/git.go

```diff
@@ -128,7 +123,10 @@ func (gr *gitResolver) resolveGitProject(ctx context.Context, gwClient gwclient.
 		llb.WithCustomNamef("[internal] GIT CLONE %s", gitURL),
 		llb.KeepGitDir(),
 	}
-	gitState := llbgit.Git(gitURL, ref, gitOpts...)
+	if keyScan != "" {
+		gitOpts = append(gitOpts, llb.KnownSSHHosts(keyScan))
```

Comment:
Should this be used in both of the `llb.Git` calls?

Reply:
ah yes, good catch. I'll fix that now.